doc/version_history.rst

```diff
@@ -9,13 +9,15 @@ Version History
 v0.20.0
 -------
 
-* Handle updated configuration and telemetry messages from low-level controller v1.1.7, which is required.
+* Handle updated configuration and telemetry messages from low-level controller 1.1.7, which is required.
```

Comment:
The version of low-level controller will be v1.1.8 if you need the `hexCmd_mic_deg` field in `telemetryStreamStructure_t`:
https://github.com/lsst-ts/ts_hexapod_controller/blob/develop/doc/versionHistory.md

Reply:
Thanks for the warning. Fixed.